com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/LuaBuilder.java

```diff
@@ -445,7 +448,11 @@ public void build(Task<Void> task) throws CompileExceptionError, IOException {
             }
         }
 
+        ResourceDigest resourceDigest = ResourceDigestCache.create(outputResource);
```

Comment:
Build a resource digest for later use when we create the archive. Add the processed Lua source and options

Reply:
One important thing to note here is that since the resource digest is based on something other than the binary data written to the archive (ie the source code and settings instead of the protobuf w. bytecode) then manifest verification will not work. The user must use `resource.store_archive(archive, cb, { verify = false })`.